### ISSUE TITLE
Suggestion: globals in package.json

### ISSUE THREAD

**Issue by Deadarius** [feross/standard#177]
It is rather uncomfortable to write something like window._ or have /* global _ */ at each .js file.
What if we have something like this inside of package.json:
```json
{
  <...>
  "standard": {
    "globals": ["_", "angular", "s"]
  }
}
```

**Comment by feross** [comment]
Agreed - this is really handy for large projects. Released as 4.5.0.

**Comment by Deadarius** [comment]
I think there is a confusion between 'globals' and 'global'
It is plural in source and test case, but singular in readme. Will it make sense to have it singular in source since it is singular /* global var1 */ ? Or is it better to just update readme.

Sorry, it must be me who confused you in my original suggestion as it says "globals" there

**Comment by feross** [comment]
I think eslint supports both `global` and `globals`, so we can support both. Released as 4.5.1.
